index.html

```diff
@@ -0,0 +1,179 @@
+    <!-- popup menu area -->
+
+    <header class="header-wrapper">
+      <div class="menu-icon hide2" id="open"><i class="fa fa-reorder"></i></div>
+    </header>
+    <main>
+      <section class="middle-section-wrapper">
+        <section class="header-resizing">
+          <h3 class="greetings">"Hello!smart world"</h3>
+          <h1 class="top-header">GLOBAL FORUM ON AI AND IoT</h1>
+        </section>
+        <section class="text-container">
+          <p class="text1">
+            A platform to discuss trends in Artificial Intelligence, Internet of
+            Things and Smart Cities and discuss the role of artificial
+            intelligence and IoT to shape smarter and more sustainable cities
+            and to address urban challenges, with the final goal of improving
+            the quality of life of people.
+          </p>
+        </section>
+        <section class="text2">
+          <h2>2022.12.3(08:30AM -08:00PM)</h2>
+          <p>
+            @Hubin Hall 3 (Second Floor - Building A), Juna Hubin Hotel Wuxi,
+            China
+          </p>
+        </section>
+      </section>
+      <div class="program">
+        <section class="program-wrapper">
+          <div class="program-header">
+            <h2>Mai<span class="inline1">n Pro</span>gram</h2>
+          </div><div class="steps">
+          <div class="step">
+            <span class="icon2"><i class="fa fa-book" aria-hidden="true"></i></span> 
+            <h4>Lecture</h4>
+            <div class="content">
+              listen to the speakers from various countries about the messages
+              of sharing and opening
+            </div>
+          </div>
+          <div class="step">
+            <span class="icon2"><i class="material-icons">burst_mode</i></span>
+            <h4>  
+              Exhibition
+            </h4>
+            <div class="content">
+              Appreciate various innovative projects from different companies
+              and startups
+            </div>
+          </div>
+          <div class="step">
+            <span class="icon2"><i class="fa fa-comments" aria-hidden="true"></i></span> 
+            <h4>Forum</h4>
+            <div class="content">
+              Have the time to share your thoughts with experts for different
+              topics
+            </div>
+          </div>
+          <div class="step">
+            <span class="icon"><i class="material-icons">build</i></span>
+            <h4>  
+              Workshop
+            </h4>
+            <div class="content">
+              Seizing the benefits of AI and IoT through data security
+            </div>
+          </div>
+          <div class="step ">
+            <span class="icon">
+              <i class="material-icons">assignment_ind</i>
+            </span>
+            <h4>
+              Info Session
+            </h4>
+            <div class="content">
+              Information on ITU Activities on "Environment, Climate Change and
```

Comment:
- [ ] Kindly make sure to follow the background color as per the original design.
- Original design:
<img width="827" alt="Screenshot 2022-04-15 at 3 29 21 PM" src="https://user-images.githubusercontent.com/70269507/163570885-8b01b215-2a04-4e96-9cdd-a850edaeac2c.png">

- Your design:
<img width="1267" alt="Screenshot 2022-04-15 at 3 29 01 PM" src="https://user-images.githubusercontent.com/70269507/163570899-f05c7ee8-76b1-4be4-b9da-8616ac08e1a6.png">